internal/rbd/replicationcontrollerserver.go

```diff
@@ -303,52 +302,33 @@ func createDummyImage(ctx context.Context, rbdVol *rbdVolume) error {
 	return nil
 }
 
-// enableImageMirroring enables the mirroring for the dummy image.
-func enableMirroringOnDummyImage(rbdVol *rbdVolume, mirroringMode librbd.ImageMirrorMode) error {
-	optName := getOperationName(rbdVol.Pool, dummyImageMirroringEnabled)
-	if _, ok := operationLock.Load(optName); !ok {
-		imgName, err := getDummyImageName(rbdVol.conn)
-		if err != nil {
-			return err
-		}
-		dummyVol := rbdVol
-		dummyVol.RbdImageName = imgName
-		// this is a idempotent call we dont need to worry multiple enable calls
-		err = dummyVol.enableImageMirroring(mirroringMode)
-		if err != nil {
-			return err
-		}
-		operationLock.Store(optName, true)
-		// Remove the dummyImageMirroringDisabled lock so that the mirroring can
-		// be disabled on the image.
-		optName = getOperationName(rbdVol.Pool, dummyImageMirroringDisabled)
-		operationLock.Delete(optName)
+// tickleMirroringOnDummyImage disables and reenables mirroring on the dummy image, and sets a
+// schedule of a minute for the dummy image, to force a schedule refresh for other mirrored images
+// within a minute
+func tickleMirroringOnDummyImage(rbdVol *rbdVolume, mirroringMode librbd.ImageMirrorMode) error {
+	imgName, err := getDummyImageName(rbdVol.conn)
+	if err != nil {
+		return err
 	}
+	dummyVol := rbdVol
+	dummyVol.RbdImageName = imgName
 
-	return nil
-}
+	dummyImageOpsLock.Lock()
+	err = dummyVol.disableImageMirroring(false)
```

Comment:
Should we ignore the error here.

Reply:
I am thinking we do not, say we ignore it, and hence the enable becomes a no-op as it was never disabled, and causes the schedule to not trigger, we would be back at square one.

Reply:
@ShyamsundarR if csi driver is restarted after disable and before enable will it be a problem? Won't we get any error here?

Reply:
Multiple disables on a disabled image is not throwing errors in the CLI, validated the same, we should be fine under that assumption.

Reply:
Yes correct. no problem here